test/empty-brace-spaces.js

```diff
@@ -0,0 +1,89 @@
+import {outdent} from 'outdent';
+import {flatten} from 'lodash';
+import {test} from './utils/test';
+
+const SPACES_PLACEHOLDER = '/* */';
+const cases = [
+	'{/* */}',
+	'function foo(){/* */}',
+	'if(foo) {/* */}',
+	'if(foo) {} else if (bar) {/* */}',
+	'if(foo) {} else {/* */}',
+	'for(;;){/* */}',
+	'for(foo in bar){/* */}',
+	'for(foo of bar){/* */}',
+	'switch (foo) {case bar: {/* */}}',
+	'switch (foo) {default: {/* */}}',
+	'try {/* */} catch(foo){}',
+	'try {} catch(foo){/* */}',
+	'try {} catch(foo){} finally {/* */}',
+	'do {/* */} while (foo)',
+	'while (foo){/* */}',
+	'foo = () => {/* */}',
+	'foo = function (){/* */}',
+	'function foo(){/* */}',
+	'foo = {/* */}',
+	'class Foo {bar() {/* */}}',
+	'foo = class {bar() {/* */}}'
```

Comment:
We also need to test for `() => {}`.

Reply:
It's already tested https://github.com/sindresorhus/eslint-plugin-unicorn/pull/933/files#diff-049935872bb22d7098f0c8cbe3b5319cf79af03b3769b1387f3d3fa20cb8a546R22